app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from app.errors import (
+    OutdatedVaccineError,
+    NotVaccinatedError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not ("vaccine" in visitor):
+            raise NotVaccinatedError()
+        elif "expiration_date" not in visitor["vaccine"]:
+            raise OutdatedVaccineError()
+        elif datetime.date.today() > visitor["vaccine"]["expiration_date"]:
+            raise OutdatedVaccineError()
```

Comment:
The code assumes that 'expiration_date' is always a valid date object. It would be safer to first check if 'expiration_date' is a valid date object before performing the comparison. This can prevent potential runtime errors if the 'expiration_date' is not in the expected format.